nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteSQL.java

```diff
@@ -93,6 +93,8 @@
 @WritesAttributes({
     @WritesAttribute(attribute="executesql.row.count", description = "Contains the number of rows returned in the select query"),
     @WritesAttribute(attribute="executesql.query.duration", description = "Duration of the query in milliseconds"),
```

Comment:
Update this description to say it's the addition of query execution and fetch time?

Reply:
Updated.